### ISSUE TITLE
Bug: Cannot search in name field from judge page [8,8,1]

### ISSUE THREAD

**Issue by rsgonzal** [Iridescent-CM/technovation-app#2852]
**Steps to Reproduce Bug**
* Log in as an admin
* Click "Judges" in the right-hand sidebar
* Fill in the field "Name or Email" with a valid first name from the table below the search form
* Click the "Search" button

**Deliverable**
* Fix this bug
* Open a PR

**Deployment Steps**
Do not deploy

### QA Steps

- Follow steps to reproduce and make sure it is working now

**Comment by viviancan** [comment]
Tested it locally - search feature working! I rebased the branch with QA to get it caught up to all the new changes on QA. I will merge it into QA once the tests have passed. Great work! Thx!

**Comment by rsgonzal** [comment]
Looks good to me!

**Comment by ophelieh** [comment]
Searched for a judge by last name, first name, and email - worked as expected.

I did notice something about location though. I saw we had 2 judges in Israel, but when I looked for judges in Israel, it only showed me one. See screenshots below.

![Screen Shot 2021-05-10 at 4 38 42 PM (2)](https://user-images.githubusercontent.com/32206421/117728081-9197c280-b1ae-11eb-953f-741c80c67af8.png)
![Screen Shot 2021-05-10 at 4 40 59 PM (2)](https://user-images.githubusercontent.com/32206421/117728086-93fa1c80-b1ae-11eb-9de0-3bc5a4431b1b.png)
